website_and_docs/content/documentation/webdriver/browsers/firefox.en.md

```diff
@@ -120,7 +120,15 @@ driver = RemoteWebDriver(options)
 
 ## Add-ons
 
-Unlike Chrome, Firefox extensions are not added as part of capabilities, they are created after starting the driver.
+Unlike Chrome, Firefox extensions are not added as part of capabilities as mentioned in
+[this issue](https://github.com/SeleniumHQ/selenium/issues/10278),
+they are created after starting the driver.
+
+The following examples are for local webdrivers.
+For remote webdrivers using Java/Ruby/C#, please refer to
+[this post](https://docs.saucelabs.com/web-apps/automated-testing/selenium/selenium4/#install-and-uninstall-add-ons).
```

Comment:
We don't want to link to vendor content in these docs. 

My plan at one point was to add *one* of those examples in https://www.selenium.dev/documentation/webdriver/drivers/remote_webdriver/ and then link to that page from the browser specific pages.

Reply:
Fixed in 36c4cb15b6408fd844b5b4445212825e7decbaa6

I opened a issue regarding browser specific functionalities: SeleniumHQ/selenium#11483, and linked to that issue instead of saucelabs.